src/jobs/release-package.yml

```diff
@@ -144,6 +143,7 @@ steps:
           - not: <<parameters.prerelease>>
           - <<parameters.sign>>
       steps:
+        - install-sfdx
```

Comment:
Only install `sfdx` when signing packages:
https://github.com/salesforcecli/plugin-release-management/blob/b7df0ac2c2936d6eb9b9c911361e93bcffb986d3/src/commands/npm/package/release.ts#L125

Reply:
what's `sfdx` being used for here?

Reply:
oh, maybe I understand...because installing the CLI includes `plugin-trust`?

Reply:
Yes! `sfdx-trust` was used here as a standalone binary, but now we need to install `sfdx` to use plugin-trust.

Reply:
I'm fine with this approach but it might be worth considering bundling `plugin-trust` with `sf-release` - that way we can speed up the build by not needing to install both `sfdx` and `sf-release`